main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/application"
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/device/deviceType"
+	"github.com/josephroberts/edge-node-manager/radio/radioType"
+)
+
+func main() {
+	exit()
+
+	nrf51822 := &device.Type{
+		Device: deviceType.NRF51822,
+		Radio:  radioType.BLUETOOTH,
+	}
+	esp8266 := &device.Type{
+		Device: deviceType.ESP8266,
+		Radio:  radioType.WIFI,
+	}
+
+	applications := make([]*application.Application, 0, 10)
```

Comment:
I think I covered this in the previous round of code review, but you don't need to do repeated appends (if you hadn't have specified a capacity it'd also be inefficient.)

You can just do `applications := []*application.Application{ ...apps... }`.